app/schema_loader/schema_loader.py

```diff
@@ -72,3 +74,10 @@ def available_s3_schemas(files):
             files.append(key.key)
     except botocore.exceptions.ClientError as e:
         logging.error("S3 error: %s", e.response['Error']['Code'])
+
+
+def validate_json_against_schema(json_to_validate):
```

Comment:
I put it in here as i thought it would be useful for when we validate the schemas on load. But i guess as its not being used now ill move it into the tests.